django-server/server/api/urls.py

```diff
@@ -1,14 +1,33 @@
 from django.urls import include, path
-from rest_framework import routers
-from . import views
+from rest_framework_nested.routers import DefaultRouter, NestedSimpleRouter
+from .views import (
+    CafeteriaViewSet,
+    FixedMenuOptionViewSet,
+    FixedMenuOptionReviewViewSet,
+    CafeteriaFixedMenuOptionReviewViewSet,
+    MenuOptionTagViewSet,
+    UploadArtifactsBetaView,
+    UploadArtifactsView)
```

Comment:
`)` in new line? That's what `black` formatter or `isort` does by default.

Reply:
Import sorting too.

Reply:
fixed